test/unit/store/model-store.js

```diff
@@ -0,0 +1,43 @@
+import ModelStore from '../../../.dist/store/model-store';
+import { assert } from 'chai';
+import { addAction } from '../../util/database';
+
+describe('ModelStore', () => {
+
+    let model;
+
+    before(() => {
+        return addAction('CLEAR_ALL_DATA');
+    });
+
+    it('should instantiate without any models', () => {
+        assert.lengthOf(ModelStore.getModels(), 0);
+    });
+
+    it('should create a model an retrieve it by the action id', () => {
```

Comment:
This description is a bit deceptive. I expect it would actually retrieve a model by address and assert it is the correct one.